modules/Bio/EnsEMBL/Variation/Utils/Sequence.pm

```diff
@@ -949,7 +949,7 @@ sub align_seqs {
 sub trim_sequences {
   my ($ref, $alt, $start, $end, $empty_to_dash, $end_first) = @_;
 
-  throw("Missing reference or alternate sequence") unless $ref && $alt;
+  throw("Missing reference or alternate sequence") unless defined $ref && defined $alt;
```

Comment:
Is  an empty alt also now accepted?
trim_sequences('A','')

Reply:
I don't think it matters for the trim_sequences call.

Reply:
Before the code change, the trim_sequences('A','') was throwing a warning. Now it will not. If does not matter then ok.

Reply:
It doesn't only throw a warning but an error and stops the whole VEP analysis. I don't think this method is the right place for allele checks.